### DISCUSSION TITLE
add PDM interpreter

### DISCUSSION THREAD

**Discussion by ramunecroft** [zed-industries/zed#6602] [Feature Requests]
### env not applied on zed

![image](https://github.com/zed-industries/community/assets/119154327/80c7754e-e465-442c-8a58-221e344162a5)

### This is my file tree

![image](https://github.com/zed-industries/community/assets/119154327/d6bef347-c36b-452a-88e5-8f62b405c4ff)

pyrightconfig.json
`{
  "venvPath": ".venv/bin/python",
  "venv": ".venv"
}
`
I thought applying the interpreter would work this way, 
but for some reason it's not being applied. Does anyone know why?

**Comment by JosephTLyons** [comment]
Hey @ramunecroft, do these settings work? Might have to restart the language server for it to be picked up.

```json
{
  "venvPath": ".",
  "venv": ".venv"
}
```

**Reply by ramunecroft** [reply]
@JosephTLyons 
Thanks! 
it's working without restart the language server!
